sources/reference/tutorials/batch.md

```diff
@@ -0,0 +1,606 @@
+provide Batch with an execution role that will allow it to pull the image from
+ECR. It will also enable Batch to output the logs of the container to CloudWatch.
+
+Don't be confused when we refer to `Elastic Container Service`, Batch is using
+it under the hood.
+
+* Navigate to the IAM console and select `Roles`
+* Select `Create role`
+* Select `Elastic Container Service` from the list
+* Select `Elastic Container Service Task` from the `Select your use case` list
+* Select `Next: Permissions`
+* In the search bar, type `AmazonECSTaskExecutionRolePolicy` and select it
+* Select `Next: Tags`
+* Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
+* Select `Next: Review`
+* Provide a name for the Role
+    * For the purpose of this guide we'll use `ctk-batch-execution-role`
+* Select `Create Role`
+
+### 8. Create your Batch Job job role
+
+The nature of `chaostoolkit-aws` means that we use `boto3` to make AWS requests
+within our experiment. To be able to make these calls, the container that is
+running our experiment needs credentials and permissions to do so.
+
+By creating a job role for our Job, we can:
+
+* Provide our Job with credentials with AWS
+* Outline exactly _what_ our Job is allowed to do
+
+* Navigate to the IAM console and select `Roles`
+* Select `Create role`
+* Select `Elastic Container Service` from the list
+* Select `Elastic Container Service Task` from the `Select your use case` list
+* Select `Next: Permissions`
+* Select `Create policy`
+* Move to the `JSON` tab and paste the following:
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeInstance*"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+* Select `Next: Tags`
+* Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
+* Select `Next: Review`
+* Provide a name for the Policy
+    * For the purpose of this guide we'll use `ctk-batch-job-policy`
+* Select `Create Policy`
+* Navigate back to the Role tab
+* In the search bar, type `ctk-batch-job-policy` and select it (You may have to
+click the refresh button)
+* Select `Next: Tags`
+* Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
+* Select `Next: Review`
+* Provide a name for the Role
+    * For the purpose of this guide we'll use `ctk-batch-job-role`
+* Select `Create Role`
+
+### 9. Create your Batch Job definition
+
+This is where we tell AWS _what_ our Job is and needs.
+
+* Navigate to the Batch console and select `Job definitions`
+* Select `Create`
+* Provide a name
+    * For the purpose of this guide we'll use `ctk-batch-job-def`
+* Leave `Fargate` selected
+* Scroll down to `Container properties`
+* For `Image`, provide the URI of your ECR image from earlier
+    * For example, when writing this guide, ours is:
+ `<our-aws-account-id>.dkr.ecr.eu-west-2.amazonaws.com/ctk-batch:latest`
```

Comment:
<your-aws-account-id> ?

Reply:
?

Reply:
Nupe, not in that context:

When writing this guide, *ours* is: